applications/ink_contract_multi-sig.md

```diff
@@ -0,0 +1,215 @@
+# Ink! Contract Multi-Sig
+- **Team Name:** Altoros LLC (dba Protofire DAO)
+- **Payment Address:** Polkadot Relay Chain: 14YXgiAqian6PWkABXGVhKB43tmbqMwscHLdhHi1qQVBeVQM
+
+## Project Overview :page_facing_up:
+### Overview
+
+To onboard developers, blockchain networks must provide developers with a set of essential building blocks. We call them a “Minimum Viable Infrastructure.” Without offering essential services, the unique capabilities of the Smart Contract Pallet are not accessible to consume. We identified this on any chain implementing contracts pallet.
+At Protofire, we specialize in removing blockers that prevent developer ecosystem organic growth. 
+In this proposal, we will address one of these blockers: the lack of a multi-sig wallet that could be edited without changing its address, providing a smooth workflow and a desirable user experience. 
+It is an essential tool for: 
+ - DAOs and individuals to manage cryptocurrency assets 
+ - Interacting with Smart Contracts in a secure manner 
+
+We have already implemented the Gnosis Safe multi-sig wallet in Astar. Still, that functionality has a narrow scope because is constrained to the EVM Pallet. So, we intend to bring similar functionalities to any substrate chain that supports Contract Pallet by implementing an Ink! smart contract with a user interface focused on the user experience.
```

Comment:
Gnosis is a great product. I like that you are very user-focused. The Polkadot ecosystem needs more of that.